### DISCUSSION TITLE
Some filenames with textual info are ignored when filename starts with (X)

### DISCUSSION THREAD

**Discussion by TrySpace** [photoprism/photoprism#4014] [General]
Currently some images and videos are named as, for example 'Info' when the label I find them in is 'Info', while there is a more informative filename available.
 
I think it is because some filesnames are ignored because the name of the file is "(1) Graph of Thing.jpg", and I think it ignores the rest and interprets the (1) as a 'garbage' name. Perhaps it should look farther than the first occurence of a 'garbage' pattern. 

#### How can we reproduce it?

1. Name image with "(0) something.gif"
2. See the name is not "something"

**Comment by graciousgrey** [comment]
PhotoPrism does not only use filenames to create titles. Titles can be generated from labels as well and it might be possible that with the current implementation we do prefer labels over parts from the filenames. 

However you will find parts of the filename in a photos keywords. So for example "(1) Graph of Thing.jpg" will get the keywords "graph" and "thing" or (0)something.jpg will get he keyword "something".
